app/src/NativeBase/Components/ColouredTag.tsx

```diff
@@ -23,20 +24,24 @@ const ColouredTag = ({ title }: ColouredTagProps) => {
   }
   const { colorMode } = useColorMode()
   const coloursToUse = colours[colorMode ?? 'light']
-  const randomColour =
-    coloursToUse[Math.floor(Math.random() * coloursToUse.length)]
+
+  const roleIndex = getRoleGroupIndex(title)
+  let selectedColour
+  if (roleIndex) {
+    selectedColour = coloursToUse[roleIndex % coloursToUse.length]
```

Comment:
Should `selectedColour` default to something -- otherwise if `roleIndex` is undefined, `selectedColour` will also be undefined and so there won't be a colour set (which I imagine we would never want to happen)?

Reply:
Would the simple thing to just default to the first item in the array?